open_spiel/python/games/domino.py

```diff
@@ -0,0 +1,346 @@
+import copy
+import itertools
+
+import numpy as np
+
+import pyspiel
+
+_NUM_PLAYERS = 2
+
+# The first player to play is the one holding the highest rank tile.
+# The rank of tiles is the following:
+#   1. Highest double.
+#   2. If none of the players hold a double, then highest weight.
+#   3. If the highest weighted tile of both players has the same weight
+#      then the highest single edge of the highest weighted tile.
+
```

Comment:
General comment: I think we should rename this game block_dominoes. Dominoes is a family of games and we may want to add more in the future.

Reply:
The more I think about this the more I'm leaning towards just leaving as is and renaming if we need to in the future. @lanctot wdyt?

Reply:
I am good with that. When I think of Dominoes this is the default version that comes to mind